engine/security/trust/trust_sandbox.md

```diff
@@ -74,65 +74,190 @@ the `trustsandbox` container, the Notary server, and the Registry server.
 
 3. Add the following to the new file.
 
-        version: "2"
+        ```yaml
```

Comment:
This yaml annotation appears on the website, and on the rendering of markdown by github. Could be due to the extra space in front.